activemq/tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+import pytest
+import requests
+
+from datadog_checks.dev import docker_run
+from datadog_checks.dev.utils import load_jmx_config
+
+from .common import HERE
+
+TEST_QUEUES = ('FOO_QUEUE', 'TEST_QUEUE')
+TEST_TOPICS = ('FOO_TOPIC', 'TEST_TOPIC')
+TEST_MESSAGE = {'body': 'test_message'}
+TEST_AUTH = ('admin', 'admin')
+
+BASE_URL = 'http://localhost:8161/api/message'
+
+
+def populate_server():
+    """Add some queues and topics to ensure more metrics are available."""
+    for queue in TEST_QUEUES:
+        url = '{}/{}?type=queue'.format(BASE_URL, queue)
+        requests.post(url, data=TEST_MESSAGE, auth=TEST_AUTH)
+
+    for topic in TEST_TOPICS:
+        url = '{}/{}?type=topic'.format(BASE_URL, topic)
+        requests.post(url, data=TEST_MESSAGE, auth=TEST_AUTH)
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    with docker_run(
+        os.path.join(HERE, 'compose', 'docker-compose.yaml'),
+        log_patterns=['ActiveMQ Jolokia REST API available'],
+        sleep=2,
+    ):
+        populate_server()
```

Comment:
This may cause issue when stopping the environment, the `dd_environment` fixture is called twice. Once during setup and once during tear down.
If you look at docker_run code you'll see that we handle set up and tear down differently.

An example of how we've done it elsewhere: https://github.com/DataDog/integrations-core/blob/master/mysql/tests/conftest.py#L35